stl/inc/random

```diff
@@ -190,8 +190,11 @@ public:
             for (_Kx = 0; _Kx < _Mx; ++_Kx) { // scramble and add any vector contributions
                 result_type _Rx1 =
                     1664525 * _Xor27(_UFirst[_Kx % _Nx] ^ _UFirst[(_Kx + _Px) % _Nx] ^ _UFirst[(_Kx - 1) % _Nx]);
-                result_type _Rx2 = static_cast<result_type>(
-                    (_Rx1 + (_Kx == 0 ? _Sx : _Kx <= _Sx ? _Kx % _Nx + _Myvec[(_Kx - 1) % _Sx] : _Kx % _Nx)) & _Mask);
+                result_type _Rx2 = static_cast<result_type>((_Rx1
+                                                                + (_Kx == 0      ? _Sx
+                                                                    : _Kx <= _Sx ? _Kx % _Nx + _Myvec[(_Kx - 1) % _Sx]
+                                                                                 : _Kx % _Nx))
+                                                            & _Mask);

```

Comment:
I am not sure if this is really an improvement

Reply:
Could we give the conditional expression a name?

Reply:
Yeah, this one is kind of a mess. I can opt-out here.

Reply:
I will totally refactor this.